src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -331,6 +344,20 @@ impl VmmService {
         Ok(VmmData::Empty)
     }
 
+    /// Get prometheus metrics.
+    fn get_metrics(&self) -> VmmRequestResult {
+        tokio::runtime::Builder::new_multi_thread()
```

Comment:
I want to call the async func get_metrics(in src/hypervisor_metrics.rs) in this synchronous function.
Here's the [blog](https://tokio.rs/tokio/topics/bridging) I was referring to at the time. And according to the official [doc](https://docs.rs/tokio/0.3.1/tokio/runtime/struct.Builder.html#current-thread-runtime-will-only-run-on-the-current-thread-via-runtimeblock_on), it will run the runtime and future on the current thread.

Reply:
Have you tested this function? I remember tokio runtimes cannot overlap. If they overlap, they panic.

Reply:
I don't know about this rule, but it seems to work correctly.

Reply:
In this PR(https://github.com/kata-containers/kata-containers/pull/5256) discussion, I finally chose to implement it as async func.

Reply:
Updated get_hypervisor/shim_metrics to be a sync func.